recipes/python/policies/policy_api_request_helper.py

```diff
@@ -0,0 +1,253 @@
+import requests
+import json
+
+content_type = "application/vnd.netbackup+json; version=4.0"
+etag = ""
+
+def perform_login(username, password, domainName, domainType, base_url):
+    url = base_url + "/login"
+    req_body = {"userName": username, "password": password, "domainName": domainName, "domainType": domainType}
+    headers = {'Content-Type': content_type}
+
+    print("Making POST Request to login for user '{}'\n".format(req_body['userName']))
+
+    resp = requests.post(url, headers=headers, json=req_body, verify=False)
+
+    if resp.status_code != 201:
+        print('Login API failed with status code {} and {}\n'.format(resp.status_code, resp.json()))
+
+    print("\nThe response code of the Login API: {}\n".format(resp.status_code))
+
+    return resp.json()['token']
+
+def create_netbackup_policy(jwt, base_url, policy_id, policy_name, policy_type):
+    url = base_url + "/config/policies/"
+    req_body = {
+        "data": {
+            "type": "policy",
+            "id": policy_id,
+            "attributes": {
+                "policy": {
+                    "policyName": policy_name,
+                    "policyType": policy_type,
+                    "policyAttributes": {},
+                    "clients": [],
+                    "schedules": [],
+                    "backupSelections": {
+                        "selections": []
+                    }
+                }
+            }
+        }
+    }
+    headers = {'Content-Type': content_type, 'Authorization': jwt}
+
+    print("\nMaking POST Request to create Policy with defaults \n")
+
+    resp = requests.post(url, headers=headers, json=req_body, verify=False)
+
+    if resp.status_code != 204:
+        print(
+            'Create Policy API with defaults failed with status code {} and {}\n'.format(resp.status_code, resp.json()))
+
+    print("\n Policy with PolicyID {}, Policy Name {}  and Policy Type {} is created with status code : {}\n".format(policy_id, policy_name, policy_type, resp.status_code))
+
+
+def get_netbackup_policies(jwt, base_url):
+    url = base_url + "/config/policies"
+    headers = {'Content-Type': content_type, 'Authorization': jwt}
+
+    print("\nMaking GET Request to list policies ")
+
+    resp = requests.get(url, headers=headers, verify=False)
+
+    if resp.status_code != 200:
+        print('List Policies API failed with status code {} and {}\n'.format(resp.status_code, resp.json()))
+
+    print("\nList policy succeeded with status code: {}\n".format(resp.status_code))
+    print("\n Json Response body for List policies  : \n{}\n".format(json.loads(resp.content)))
```

Comment:
What's printed isn't the JSON response. It's the stringification of the Python dict representing the object described in the response.

```suggestion
    print("JSON response body for List policies:\n{}".format(resp.content))
```

The key point is that if you want to see the JSON response, then you should print that, not pass it through a parser and a formatter first, especially when the formatter doesn't print JSON.